mitxgraders/baseclasses.py

```diff
@@ -98,19 +98,20 @@ def __call__(self, expect, student_input):
         Notes:
             This function ignores the value of expect.
```

Comment:
I think some more rewording is necessary here...

Reply:
I just reworded the comments a bit. I left the part about `expect` being ignored: It is true that `AbstractGrader.__call__` completely ignores `expect`. I agree this is a little weird... feel free to word something better :)